docs/self-styling/configuration-page.md

```diff
@@ -64,35 +67,39 @@ The `delegation` directory contains resources that allow customizing the web exp
     | **Username**                   | The username for the git repository.
     | **Password**                   | The password for the git repository. Instead of using a password, you can generate a GitHub personal access token here: [Personal access tokens](https://github.com/settings/tokens).
 
-2. Click [Next: select reference]. (*The form has been validated and stored.*) 
+6. Click [Save]. (*The form has been validated and stored.*) 
 
 ### Select reference
 
-1. After completing the steps under [Create Git repository connection](#create-git-repository-connection), the *Select a Reference*-window opens: <br> <br>
-![Page to select a reference](./images/Selectareference.png)
+1. After completing the steps under [Create Git repository connection](#create-git-repository-connection), the *Synchronization*-window opens: <br> <br>
+![Page to select a reference](./images/selectreference.png)
+   | Field                          | Description
+   |--------------------------------|------------
+   | **Synchronisation mode**       | Allows selecting synchronization mode. Development mode allows selecting branches and tags from the configured repository, but does not cache the data. Production mode allows selecting tags only, but caching is enabled.
```

Comment:
A number of switching between the z and s spelling of synchronisation. The actual implementation seems to use the 's' version so lets go with that.